dispatch/FileCache.h

```diff
@@ -60,6 +61,12 @@
 // an LRU cache. jhrg 11/03/23
 #define FORCE_ACCESS_TIME_UPDATE 1
 
+// I tried using fcntl(2) because it's supposedly better than flock(2) when using
+// NFS, but I think that is really for NFS 3. For NFS 4, flock(2) is supposed to be
+// fully supported. It's much easier and it supports multi threading. The fcntl(2)
+// code here is not yet passing all the tests!
+#define USE_FCNTL 0
```

Comment:
So if this works with flock() will this be something that goes away??

Reply:
Yes. I thought I got that. I just made another push that tightens up the unit tests and I think I got this then as well.

Reply:
Arrrgh, I did not. but I got it now.